openapi/data_repository_service.swagger.yaml

```diff
@@ -350,7 +352,9 @@ definitions:
           A name declared by the bundle author that must be
           used when materialising this object, 
           overriding any name directly associated with the object itself.
-          This string MUST NOT contain any slashes.
+          If this is a content object inside of a bundle then the name must be unique within that bundle. 
```

Comment:
We *know* this is a content object inside a bundle -- maybe simplify to "The name must be unique with the containing bundle."

Reply:
yep I like that